onnxruntime/core/mlas/lib/quantize.cpp

```diff
@@ -210,6 +210,92 @@ Return Value:
     }
 }
 
+template
+void
+MLASCALL
+MlasQuantizeLinearKernel<int8_t>(
+    const float* Input,
+    int8_t* Output,
+    size_t N,
+    float Scale,
+    int8_t ZeroPoint
+    );
+
+template
+void
+MLASCALL
+MlasQuantizeLinearKernel<uint8_t>(
+    const float* Input,
+    uint8_t* Output,
+    size_t N,
+    float Scale,
+    uint8_t ZeroPoint
+    );
+
+void
+MLASCALL
+MlasQuantizeLinearS8Kernal(
+    const float* Input,
+    int8_t* Output,
+    size_t N,
+    float Scale,
+    int8_t ZeroPoint
+    )
+{
+    MlasQuantizeLinearKernel<int8_t>(Input, Output, N, Scale, ZeroPoint);
+}
+
+void
+MLASCALL
+MlasQuantizeLinearU8Kernal(
+    const float* Input,
+    uint8_t* Output,
+    size_t N,
+    float Scale,
+    uint8_t ZeroPoint
+)
+{
+    MlasQuantizeLinearKernel<uint8_t>(Input, Output, N, Scale, ZeroPoint);
+}
+
+template<>
+void
+MLASCALL
+MlasQuantizeLinear<int8_t>(
+    const float* Input,
+    int8_t* Output,
+    size_t N,
+    float Scale,
+    int8_t ZeroPoint
+    )
+{
+#if defined(MLAS_TARGET_AMD64)
+    MlasPlatform.QuantizeLinearS8Kernel(
+#else
+    MlasQuantizeLinearS8Kernal(
```

Comment:
Kernal->Kernel. Can the wrapper from above just be inlined here instead (MlasQuantizerLinear<int8_t>).

Reply:
Nevermind, I see why you were doing this (the platform.cpp reference). platform.cpp could just reference the template too, but I guess this is consistent with what's been done in qladd.cpp etc and could be changed later.